R/calculate_stay.R

```diff
@@ -51,6 +51,7 @@ calculate_stay <- function(year, start_date, end_date, sc_qtr = NULL) {
       lubridate::period(1L, "days")
     )
 
+    # check logic here for care home methodology
     dummy_end_date <- dplyr::case_when(
```

Comment:
Check care home logic here and decide if we need the quarter calculation functions